### PR TITLE
Update set-up-node-with-installer.md

### PR DESCRIPTION
**Pull request by dubrado**
Just some small edits to the language and format in some areas to clean things up for the newer node builders with less experience.

### CODE REVIEW

File: docs/nodes/build/set-up-node-with-installer.md [hunk 1]
```diff
@@ -13,17 +13,16 @@ installs](#unattended-installation).
...
 
 - CPU: Equivalent of 8 AWS vCPU
 - RAM: 16 GiB
 - Storage: 1 TiB
 - OS: Ubuntu 20.04 or MacOS &gt;= 12
 - Network: sustained 5Mbps up/down bandwidth
 
-Please note that HW requirements shall scale with the amount of AVAX staked on
-the node. Nodes with big stakes (100k+ AVAX) will need more powerful machines
+NOTE: HW requirements shall scale with the amount of AVAX staked on
```
Comment (yulin-dong) [inline docs/nodes/build/set-up-node-with-installer.md:24]: Should docusaurus admonition https://docusaurus.io/docs/markdown-features/admonitions be used here?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/nodes/build/set-up-node-with-installer.md [hunk 2]
```diff
@@ -177,14 +176,15 @@ be prompted:
...
-requests on any network interface you will need to set up a firewall to only let
-through RPC requests from known IP addresses, otherwise your node will be
-accessible to anyone and might be overwhelmed by RPC calls from malicious
-actors! If you do not plan to use your node to send RPC calls, enter `local` for
-increased node security.
+`local` This setting only allows RPC requests from the node machine itself.
+`any` This setting allows you to send RPC requests to your node
+from a remote machine. 
+
+NOTE: If you choose to allow RPC requests on any network interface you will need
```
Comment (yulin-dong) [inline docs/nodes/build/set-up-node-with-installer.md:183]: admonition

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Comment by cgcardona** [comment]
Please merge the latest from `master` into your PR.

**Review by cgcardona — DISMISSED** [review]
LGTM once you merge `master` into your branch and after the requested admonitions are in place.

File: docs/nodes/build/set-up-node-with-installer.md (hunk 1 above)
Comment (yulin-dong) [inline docs/nodes/build/set-up-node-with-installer.md:24]:
```suggestion
::: note
 HW requirements shall scale with the amount of AVAX staked on
```

add the closing :::

**Review by yulin-dong — CHANGES_REQUESTED** [review]

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**